README.md

```diff
@@ -0,0 +1,65 @@
+# An Alexa Smart Home App for openHAB 2
+
+This is a Amazon Alexa Smarthome skill for openHAB 2.    
+
+# General Installation Instructions
+## Requirements
```

Comment:
please add an empty line after every headline and before and after every code section as otherwise Jekyll will mess up the formatting.